### PR TITLE
Update safety-scanner-download.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed windows server 2008 as it is not working and errors out stating that the operating system is not supported

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9733

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9733

File: microsoft-365/security/intelligence/safety-scanner-download.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Microsoft Safety Scanner is a scan tool designed to find and remove malware from
 
 ## System requirements
 
-Safety Scanner helps remove malicious software from computers running Windows 11, Windows 10, Windows 10 Tech Preview, Windows 8.1, Windows 8, Windows 7, Windows Server 2019, Windows Server 2016, Windows Server Tech Preview, Windows Server 2012 R2, Windows Server 2012, Windows Server 2008 R2, or Windows Server 2008. For details, refer to the [Microsoft Lifecycle Policy](/lifecycle/).
+Safety Scanner helps remove malicious software from computers running Windows 11, Windows 10, Windows 10 Tech Preview, Windows 8.1, Windows 8, Windows 7, Windows Server 2019, Windows Server 2016, Windows Server Tech Preview, Windows Server 2012 R2, Windows Server 2012 or Windows Server 2008 R2. For details, refer to the [Microsoft Lifecycle Policy](/lifecycle/).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/intelligence/safety-scanner-download.md]:
```suggestion
Safety Scanner helps remove malicious software from computers running Windows 11, Windows 10, Windows 10 Tech Preview, Windows 8.1, Windows 8, Windows 7, Windows Server 2019, Windows Server 2016, Windows Server Tech Preview, Windows Server 2012 R2, Windows Server 2012, or Windows Server 2008 R2. For details, refer to the [Microsoft Lifecycle Policy](/lifecycle/).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@yogkumgit Validation has been passed.

@Dansimp Please merge the PR

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder

**Pull request merged**